File: 백준/Bronze/10950. A＋B － 3/A＋B － 3.cc
#include <iostream>
using namespace std;
int main(void)
{
    int count, a, b, i, j;
    cin >> count;
    int c[256];
    for (i = 0; i < count; i++)
    {
        cin >> a >> b;
        c[i] = a + b;
    }
    for (j = 0; j < count; j++)
    {
        cout << c[j]<<endl;
    }
}

File: 백준/Bronze/10950. A＋B － 3/A＋B － 3.java
import java.util.Scanner;
public class Main{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        int c[] = new int[256];
        int count=sc.nextInt();
        for(int i=0;i<count;i++){
            int a=sc.nextInt();
            int b=sc.nextInt();
            c[i] = a + b;
        }
        for (int j = 0; j < count; j++)
        {
            System.out.println(c[j]);
        }
    }
}